docs/source/ens.rst

```diff
@@ -39,11 +47,19 @@ Use :code:`createSubDomain` function for creating a sub domain:
 
   await ensBuilder.createSubDomain('ethworks.test');
 
-Also, it's possible to create a sub domain recursively, if the top domain doesn't exist, by specifying the appropriate option:
+.. tabs::
 
-.. code-block:: ts
+  .. group-tab:: Waffle 3.0.0
 
-  await ens.createSubDomain('waffle.ethworks.tld', {recursive: true});
+    It is also possible to create a sub domain recursively, if the top domain doesn't exist, by specifying the appropriate option:
```

Comment:
Instead of tabs, let's just add a note that feature was introduced in Waffle 3.0 at the top of the file.
#notabswhenunsupported